src/services/overpassSearch.ts

```diff
@@ -47,8 +47,17 @@ const convertOsmIdToMapId = (apiId: OsmId) => {
   return parseInt(`${apiId.id}${osmToMapType[apiId.type]}`, 10);
 };
 
+type OverpassGeoJson = Feature & {
+  osmMeta: {
+    type: string;
+    id: number;
```

Comment:
Is this really needed? `Feature` already has osmMeta with type:string+id:number. It was changed recently.

Reply:
I needed it while debugging and it seems like I forgot to remove it